plugins/test/unit/plugins/importers/yum/test_upload.py

```diff
@@ -651,6 +664,59 @@ def test__generate_rpm_data_sanitizes_checksum_type(self):
         self.assertEqual(metadata['vendor'], None)
 
 
+class UploadDRPMPackageTests(unittest.TestCase):
+    """
+    tests upload of DRPM package
+    """
+    def setUp(self):
+        super(UploadDRPMPackageTests, self).setUp()
+
+        sample_rpm_filename = os.path.join(DATA_DIR,
```

Comment:
this testcase is about drpm and not rpm, so please update variable names respectively